libraries/chain/market_evaluator.cpp

```diff
@@ -168,19 +168,8 @@ void_result call_order_update_evaluator::do_evaluate(const call_order_update_ope
 
    _dynamic_data_obj = &_debt_asset->dynamic_asset_data_id(d);
 
-   /***
-    * We have softfork code already in production to prevent exceeding MAX_SUPPLY between 2018-12-21 until HF 1465.
-    * But we must allow this in replays until 2018-12-21. The HF 1465 code will correct the problem.
-    * After HF 1465, we MAY be able to remove the cleanup code IF it never executes. We MAY be able to clean
-    * up the softfork code IF it never executes. We MAY be able to turn the hardfork code into regular code IF
-    * noone ever attempted this before HF 1465.
-    */
-   if (next_maintenance_time <= SOFTFORK_CORE_1465_TIME)
-   {
-      if ( _dynamic_data_obj->current_supply + o.delta_debt.amount > _debt_asset->options.max_supply )
-         ilog("Issue 1465... Borrowing and exceeding MAX_SUPPLY. Will be corrected at hardfork time.");
-   }
-   else
+   // We cannot remove this, as there were (at least 3) instances of this happening before the hardfork
```

Comment:
But they got corrected later, so we can remove the one-time processing code? Please leave a comment.

Reply:
Replays with blocks before the hf will still need this I believe.

Reply:
It looks as if at hardfork time, current_supply was below max_supply on all assets. I am verifying. If so, the one-time processing code can be removed. I am also getting very tired of abitmore always being right.